egs/hub4_english/s5/local/format_data.sh

```diff
@@ -0,0 +1,133 @@
+#! /bin/bash
+
+# Copyright 2016  Vimal Manohar
+# Apache 2.0.
+
+set -e
+set -o pipefail
+
+echo "$0 $@"  # Print the command line for logging
+
+noise_word="<NOISE>"
+spoken_noise_word="<SPOKEN_NOISE>"
+
+. utils/parse_options.sh || exit 1;
+
+. ./path.sh || exit 1;
+
+if [ $# -ne 0 ]; then
+  echo "Usage: $0"
+  exit 1
+fi
+
+srcdir=data/local/data
+tmpdir=data/local/
+
+export PATH=$PATH:$KALDI_ROOT/tools/sph2pipe_v2.5
+
+###############################################################################
+# Format 1996 English Broadcast News Train (HUB4)
+###############################################################################
+mkdir -p data/train_bn96
+
+local/data_prep/format_1996_bn_data.pl \
+  $srcdir/train_bn96/audio.list $srcdir/train_bn96/transcripts.txt \
+  data/train_bn96 || exit 1
+
+mv data/train_bn96/text data/train_bn96/text.unnorm
+local/normalize_transcripts.pl $noise_word $spoken_noise_word \
+  < data/train_bn96/text.unnorm > data/train_bn96/text
+
```

Comment:
I inspected the normalized output, and noticed there are more cases which need careful normalization:
1. Obviously there's a "^" at the beginning of every previously capitalized word: ^nichols, which needs to be removed. 2. We'd better map acronyms like _f_b_i to fbi (within CMUDict's vocab) or the fisher convention: f._b._i._ (within swbd lexicon's vocab), or three separate characters f. b. i.   @danpovey which convention do you prefer? The choice puzzles me a lot as I'm working data normalization issues with the multi_en recipe.